api/registry/registry_integration.go

```diff
@@ -32,14 +32,28 @@ func (d *idm) Init(ctx types.Context, config gofig.Config) error {
 	}
 	d.Context = ctx
 	d.used = map[string]int{}
-	d.WithField("pathCache", "").Debug("checking volume path cache setting")
+	ctx.WithField(types.ConfigVolPathCache, d.pathCache()).Info(
+		"empty path for unused volume setting")
+	ctx.WithField(types.ConfigVolUnmountIgnoreUsed, d.ignoreUsedCount()).Info(
+		"ignore used count on unmount setting")
+	ctx.WithField(types.ConfigVolMountPreempt, d.preempt()).Info(
+		"pre-emptive mount setting")
+	ctx.WithField(types.ConfigVolCreateDisable, d.disableCreate()).Info(
+		"volume creation disabled setting")
+	ctx.WithField(types.ConfigVolRemoveDisable, d.disableRemove()).Info(
+		"volume removal disabled setting")
+
```

Comment:
If you have a bunch of settings in a row, wouldn't it be cleaner to do something
like this?

```
ctx.WithFields(log.Fields{
    types.ConfigVolPathCache: d.pathCache(),
    types.ConfigVolUnmountIgnoreUsed: d.ignoreUsedCount(),
    types.ConfigVolMountPreempt: d.preempt(),
    types.ConfigVolCreateDisable: d.disableCreate(),
    types.ConfigVolRemoveDisable: d.disableRemove(),
}).Info("integration driver settings")
```

Because, I have to say, textual descriptions of configuration keys have no place in logs like this. Not like this anyway. If information about a setting/key needs to be provided to a user/developer it should be either in the documentation or as an extension of the key's constant with some method `Doc() string`. 

See the `ContextKey` type and [the way it performs lookups](https://github.com/emccode/libstorage/blob/master/api/types/types_context.go#L116) for its text representations (although in my case I should be using `stringer` to generate code, but you get the point).